### ISSUE TITLE
Add Identity and Signature Management

### ISSUE THREAD

**Issue by nashv** [whiteout-io/mail-html5#263]
Currently, Whiteout.io provides no options to change identities or signatures. Worse, it sticks in a one line default signature-ad making it dicy to use for professional email.

**Comment by tanx** [comment]
Thanks for your feedback. We're planning to add support for editing removing the signature.

**Comment by Mosrite** [comment]
What about identity management? I use my main e-mail address with different aliases/identities and I have separate PGP-keys for each one. Therefore, I cannot really use Whiteout at the moment without missing out on some features of e-mailing that are critical for me. Will identity management be added in the future?

**Comment by tanx** [comment]
Yes. Multi account is coming. See https://github.com/whiteout-io/mail-html5/issues/100

**Comment by Mosrite** [comment]
Thanks. I'm not an expert, so I'm not sure if we are talking about the same thing here. I would understand "multi account" as using several different mail accounts (probably with address from different providers) in one client, e.g. private mails and work-related mails.
What I mean are different identities that are linked to one e-mail address. You can set up aliases, so you can basically send and receive mails with an address that is not identical to your original address, but uses the same in and outbox.
https://en.wikipedia.org/wiki/Email_alias

**Comment by tanx** [comment]
Ah ok. Gotcha. That seems like a reasonable feature request.

**Comment by Pazuu** [comment]
First: I love whiteout.io and your product is wonderful! 
But as i am telling people every day how great encryption works, its really annoying for them, that there is moreover "encrypting advertisement" below every of my e-mails. 
I would love to choose my own signature below the text. :o) 
Thank you so much for your work!!
